docs/content/release-notes/flink-1.15.md

```diff
@@ -0,0 +1,379 @@
+itself shaded in the table uber jar). 
+[FLINK-24687](https://issues.apache.org/jira/browse/FLINK-24687) moved 
+`flink-connector-files` 
+out of the `flink-table` uber jar. In order to address this, Flink 1.15 implements
+a combined approach for both making usage of `flink-connector-base` consistent 
+and ensuring a smooth transition for both Flink users and for external connector
+developers:
+
+1. all internal Flink connectors that depend on `flink-connector-base` now shade
+   and relocate it.
+2. for compatibility, until external developers implement the same change,
+   flink-connector-base is also included into `flink-dist`.
+3. It is recommended for the external connector developers to shade and relocate
+   `flink-connector-base` in their implementations.
+
+### [Merge flink-connector-testing into flink-connector-test-utils (FLINK-25712)](https://issues.apache.org/jira/browse/FLINK-25712)
+
+The `flink-connector-testing` module has been removed and should use
+`flink-connector-test-utils` module instead.
+
+### [Support to disable caching missing key for lookup cache (FLINK-24861)](https://issues.apache.org/jira/browse/FLINK-24861)
+
+You can now configure the JDBC connector to enable or disable caching results 
+for keys which don't exist in the database. This is enabled by default.
+
+
+### [Support partition keys through metadata (for FileSystem connector) (FLINK-24617)](https://issues.apache.org/jira/browse/FLINK-24617)
+
+Now the formats implementing `BulkWriterFormatFactory` don't need to implement
+partition keys reading anymore, as it's managed internally by `FileSystemTableSource`.
+
+## Runtime & Coordination
+
+### [Implement file-based JobResultStore (FLINK-25431)](https://issues.apache.org/jira/browse/FLINK-25431)
+
+Introduces a file-based version of the `JobResultStore` with new configuration
+parameters `job-result-store.storage-path` and `job-result-store.delete-on-commit`. 
+This feature persists the final state of a job including its state of cleanup.
+
+### [Integrate retry strategy for cleanup stage (FLINK-25433)](https://issues.apache.org/jira/browse/FLINK-25433)
+
+Adds retry logic to the cleanup steps of a finished job. This feature changes the
+way Flink jobs are cleaned up. Instead of trying once to clean up the job, this
+step will be repeated until it succeeds. Users are meant to fix the issue that
+prevents Flink from finalizing the job cleanup. The retry functionality can be
+configured and disabled. More details can be found 
+[in the documentation](https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/deployment/config/#retryable-cleanup).
+
+### [Introduce common interfaces for cleaning up local and global job data (FLINK-25432)](https://issues.apache.org/jira/browse/FLINK-25432)
+
+Utilizes the newly added component `JobResultStore` (see 
+[FLINK-25431](https://issues.apache.org/jira/browse/FLINK-25431)) to pick up
+already finished but not properly cleaned up jobs after a failover.
```

Comment:
This is an internal thing that doesn't belong into the release notes afaik. @XComp

Reply:
You're actually right. The important things are the retry configuration (FLINK-25433) and the file-based JRS (FLINK-25431). I will remove the release notes from FLINK-25432.

Reply:
Done... ✅